src/eduid_webapp/reset_password/views/reset_password.py

```diff
@@ -102,6 +98,8 @@
 from eduid_webapp.reset_password.helpers import send_verify_phone_code
 from eduid_webapp.reset_password.app import current_reset_password_app as current_app
 
+SESSION_PREFIX = "eduid_webapp.reset_password.views"
```

Comment:
What is this?

Reply:
It comes from [here](https://github.com/SUNET/eduid-webapp/blob/master/src/eduid_webapp/actions/actions/mfa.py#L62) as a prefix for the session. The idea is to make a session namespace for the service - it's in the queue